Dockerfile

```diff
@@ -32,12 +32,10 @@ RUN echo "Building workspace with feature(s) '$CARGO_FEATURES' and profile '$CAR
     && mkdir -p /quickwit/bin \
     && find target/$CARGO_PROFILE -maxdepth 1 -perm /a+x -type f -exec mv {} /quickwit/bin \;
 
-# Change the default configuration file in order to make the searcher
-# and indexer services accessible from outside of docker.
+# Change the default configuration file in order to make the rest,
+# grpc servers and gossip accessible from outside of docker.
 COPY ./config/quickwit.yaml ./config/quickwit.yaml
-RUN sed -i 's/#indexer/indexer/g' ./config/quickwit.yaml \
-    && sed -i 's/#searcher/searcher/g' ./config/quickwit.yaml  \
-    && sed -i 's/#  rest_listen_address: 127.0.0.1/ rest_listen_address: 0.0.0.0/g' ./config/quickwit.yaml
+RUN sed -i 's/#listen_address: 127.0.0.1/listen_address: 0.0.0.0/g' ./config/quickwit.yaml
```

Comment:
@evanxg852000 I really would prefer to copy a file instead of playing with `sed` :)

Reply:
ok let's have a docker dedicated cinfig file then.
Should I add update it in my PR?

Reply:
We could do that after the release, what do you think?

Reply:
I think it is ok though. We could really use host interpolation in the conf though. :-S

0.0.0.0 seems a tiny bit dangerous.
What do official image do? Should we listen to 172.17.42.1 for instance?

Reply:
I mostly see images with app server binding to `0.0.0.0` within docker (vector, Postgres). probably because it's just easy to run the image this way. I think the relational is docker is ran within a host and should take care of closing the gate to the world. The user can still change that.

Reply:
let's keep sed then.